docs/Grade.md

```diff
@@ -0,0 +1,58 @@
+# Grade Component
+
+The `Grade` component is a customizable avatar-based component that represents different grades. It allows you to display a label with a specified background color and font color.
+
+## Usage
+
+```tsx
+import { Grade } from '@brightlayer-ui/react-native-components';
+
+<Grade label={'CS'} fontColor="#FFFFFF" backgroundColor="#198900" size={40} />;
+
+```
+
+### Grade API
+
+| Prop Name           | Description                                | Type                                   | Required | Default                |
+| ------------------- | ------------------------------------------ | -------------------------------------- | -------- | ---------------------- |
+| label               | The Text shown in Grade                    | String                                 | yes      |                        |
+| fontColor           | The color of the text label.               | string                                 | yes      |                        |
+| backgroundColor     | The Background color of Grade              | string                                 | yes      |                        |
+| size                | The radius of Grade component              | number                                 | yes      |                        |
+
+
+## SubComponent
+
+The Subcomponents gives you a set of preconfigured components that can directly be used 
```

Comment:
Again, letter casing. And you may want to consider running the text via a grammar checker.